main.js

```diff
@@ -75,13 +74,14 @@ function createWindow () {
     // Workaround for menu appearing (https://github.com/electron/electron/issues/16521)
     if(!openDev)
         electron.Menu.setApplicationMenu(null);
-    
+
     // Create the browser window.
     mainWindow = new BrowserWindow({
-        width: 1200 + (openDev ? 700 : 0), height: 700,
+        width: 950 + (openDev ? 700 : 0), height: 600,
         // This would disable the system title bar and frame window
         // so if this is false we need a custom window top bar
         frame: true,
+        resizable: false,
```

Comment:
Currently the layout kinda breaks when resized, until its fixable i think the window should be unresizable temporarily

Reply:
That's acceptable. I'll just need to remember to create an issue for that if this isn't fixed before merging.

Reply:
I will try to work on this

Reply:
I think it would be nice to always allow resizing when in development mode, as the hard-coded value here makes it hard to use the Chrome Developer Tools for debugging the UI.